kernel/byterun/dune

```diff
@@ -4,7 +4,7 @@
  (public_name coq-core.vm)
  (foreign_stubs
   (language c)
-  (names coq_fix_code coq_float64 coq_memory coq_values coq_interp)
+  (names coq_fix_code coq_memory coq_values coq_interp)
```

Comment:
`coq_float64` is used in the VM:
```
Error: Error on dynamically loaded library: ./kernel/byterun/dllcoqrun_stubs.so: ./kernel/byterun/dllcoqrun_stubs.so: undefined symbol: coq_next_up
```

Reply:
Sure, I just mean that the VM (kernel/byterun/coq_interp.c) needs to somehow have access to the content of the C file. I'm unfortunately not fluent enough in dune to tell you the best way to achieve this.

Reply:
That's a linking error, not a compile-time one.